FolderRouter.ts

```diff
@@ -0,0 +1,349 @@
+import express = require("express");
```

Comment:
Are these the exact same routes just split into multiple files?

Reply:
yes

Reply:
although with some added routes, as discussed